src/glossary.rst

```diff
@@ -1759,13 +1759,15 @@ Glossary
          * :ref:`Runahead Limiting`
          * :term:`active cycle`
 
+   skip mode
+      A task run mode that allows setting tasks to run instantly.
```

Comment:
They don't "run instantly" so much as skip running but complete outputs. Not sure how best to put that succinctly. Maybe this?:

```suggestion
      A task run mode that skips execution but completes outputs as if the task had run.
```